langchain_linkup/search_retriever.py

```diff
@@ -31,7 +32,14 @@ class LinkupSearchRetriever(BaseRetriever):
         linkup_api_key: Optional[str] = None
             The API key for the Linkup API. If None (the default), the API key will be read from
             the environment variable `LINKUP_API_KEY`.
-
+        from_date: Optional[date] = None
```

Comment:
If it's a `date`, it's not really in iso-format (this remark is true in the whole PR since they are a lot of copy pastes)